numerics/root_finders_test.cpp

```diff
@@ -54,7 +57,134 @@ TEST_F(RootFindersTest, SquareRoots) {
     } else {
       EXPECT_THAT(evaluations, AllOf(Ge(49), Le(58)));
     }
+
+    evaluations = 0;
+    EXPECT_THAT(Brent(equation, t_0, t_max) - t_0,
+                AlmostEquals(Sqrt(n / si::Unit<Acceleration>), 0, 1));
+    EXPECT_THAT(evaluations, AllOf(Ge(7), Le(15)));
+  }
+}
+
+TEST_F(RootFindersTest, WilkinsGuFunction) {
+  // This test constructs a pathological function for which Brent is quadratic.
+  // The construction follows [WG13] and the notation therein.
+  // We will search for a root of the constructed function over [a, b].
+  double const a = 1;
+  double const b = 2;
+
+  // We first construct the sequence of points X at which Brent will be made to
+  // evaluate the function; see section 4.1.
+  std::vector<double> X;
```

Comment:
The name `X` doesn't appear in section 4.1.  Maybe reference the end of section 4.2 after equation (9).